### ISSUE TITLE
Add syntax highlighting for TypeScript code blocks

### ISSUE THREAD

**Issue by nventuro** [OpenZeppelin/docs.openzeppelin.com#60]
Code blocks marked as `typescript` using triple backticks don't get stylized.

This is what the HTML looks like:

```html
<code class="language-typescript hljs" data-lang="typescript">... </code>
```

**Comment by frangio** [comment]
We need to change this file: https://github.com/OpenZeppelin/docs.openzeppelin.com/blob/master/ui/src/scripts/vendor/highlight.js

Can you submit a PR please?

**Comment by frangio** [comment]
Fixed in #61.
